galley/pkg/config/source/inmemory/collection.go

```diff
@@ -52,6 +53,13 @@ func (c *Collection) Start() {
 	defer c.mu.Unlock()
 	c.synced = true
 
+	// If this collection is disabled, just sent a Disabled event and then a full sync.
+	if c.disabled {
+		c.dispatchEvent(event.DisabledFor(c.collection))
```

Comment:
Why send both? Doesn't disabled imply FullSync semantics?

Reply:
Got rid of this, obsolete.